packages/location-field/src/index.tsx

```diff
@@ -439,6 +439,7 @@ const LocationField = ({
           bgColor={layerColorMap.stations}
           centeredText
           header
+          role="group"
```

Comment:
The element with the `group` role should be a container with the items of the group contained within. If not doing containing elements, an alternate approach is to say `role="heading"` for these items, akin to bootstrap's implementation.